source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,132 @@
 
 Latest Mattermost Releases:
+- [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
 - [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
-- [Release v6.2 - Feature Release](#release-v6-2-feature-release)
 - [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
 
+## Release v6.6 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2022-04-16**
+
+### Compatibility
+ - Updated Safari recommended minimum version to v14.1+.
+
+### Important Upgrade Notes
+ - The Apps Framework protocol for binding/form submissions has changed, by separating the single `call` into separate `submit`, `form`, `refresh` and `lookup` calls. If any users have created their own Apps, they have to be updated to the new system.
+ - The default for ``ThreadAutoFollow`` has been changed to ``true``. This does not affect existing configurations where this value is already set to ``false``.
+
+**IMPORTANT:** If you upgrade from a release earlier than v6.5, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Apps Framework 1.0: General Availability
+ - The App Framework allows developers to build integrations with Mattermost that seamlessly work across Mattermost’s desktop and mobile clients, can be coded in any language developers are comfortable with.
+
+#### Actions Restructure
+ - The **Actions** menu was restructured to reduce clutter.
+
+#### Nested Permalink Previews
+ - Added nested previews for permalinks.
+
+#### Playbook Updates
+ - 
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a right-hand side Channel Info panel to see and interact with channel information.
+ - Added support for inline editing of posts.
+ - Changed the Mattermost indigo theme to match the dark theme in code blocks.
+ - Updated in-product links to legacy domain about.mattermost.com.
+ - Added a copy button when hovering over code blocks.
+ - Added a post menu item to copy raw text.
+ - Added a loading indicator to the **Threads** global list each time more posts are fetched on infinite scroll.
+ - Added search guidance to the **Threads** global list when no more posts can be loaded. This is only shown if you’ve scrolled to load older posts and reach the end of the list.
+ - Added accessibility support for custom statuses.
+ - Tooltip is now only displayed when text is too long in the announcement banner.
+ - When restricting direct messages to users on the same team, bots are now excluded from that restriction.
+ 
+#### Performance
+ - Improved performance when clearing notifications with Collapsed Reply Threads enabled.
+ - Improved performance of Collapsed Reply Threads when backend is enabled but frontend is disabled.
```

Comment:
Improved performance of Collapsed Reply Threads when `ThreadAutoFollow` is enabled but `CollapsedThreads` is disabled.